setup.py

```diff
@@ -502,10 +502,10 @@ def get_sphinx_theme_version() -> str:
     'pre-commit',
     'pylint==2.6.0',
     'pysftp',
-    'pytest',
+    'pytest~=6.0',
```

Comment:
We are already using the latest pytest:

https://github.com/apache/airflow/blob/ae2e0275bde2e551a63e2ee9b1c56538535e7657/constraints-3.6.txt#L327

https://github.com/apache/airflow/blob/ae2e0275bde2e551a63e2ee9b1c56538535e7657/constraints-3.7.txt#L322

Reply:
In constraints yes, this enforces it for anyone now using the constraint files too.